packages/wdio-cli/src/launcher.ts

```diff
@@ -238,6 +239,24 @@ class Launcher {
         })
     }
 
+    /**
+     * Format the specs into an array of objects with files and retries
+     */
+    formatSpecs(capabilities: (Capabilities.DesiredCapabilities | Capabilities.W3CCapabilities | Capabilities.RemoteCapabilities), specFileRetries: number) {
+        let files: (string | string[])[] = []
+        let returnValue: WorkerSpecs[] = []
+
+        files = this.configParser.getSpecs((capabilities as Capabilities.DesiredCapabilities).specs, (capabilities as Capabilities.DesiredCapabilities).exclude)
+        files.forEach(file => {
```

Comment:
no need to create a variable and push items, simply map the files and return them

Reply:
OK